Cython/Compiler/Nodes.py

```diff
@@ -625,6 +625,7 @@ class CFuncDeclaratorNode(CDeclaratorNode):
     # has_varargs      boolean
     # exception_value  ConstNode or NameNode    NameNode when the name of a c++ exception conversion function
     # exception_check  boolean or "+"    True if PyErr_Occurred check needed, "+" for a c++ check
+    # exc_clause       boolean    True if exception clause is explicitly declared
```

Comment:
I don't really like the name - maybe `has_explicit_exc_clause`?

I also have a vague memory we already have this information somewhere - I'll try to have a look later. I could be misremembering though.

Reply:
Parser seems to not pass further exc_clause:
https://github.com/cython/cython/blob/e9b08d3f071ebfad7f4886e43173cd0f72df6676/Cython/Compiler/Parsing.py#L2960-L2976